sources/packages/backend/apps/db-migrations/src/sql/Reports/Update-daily-disbursement-file.sql

```diff
@@ -0,0 +1,98 @@
+UPDATE
+    sims.report_configs
+SET
+    report_sql = (
+        'WITH disbursement_receipt_dataset AS (
+            SELECT
+                *
+            FROM
+                sims.disbursement_receipts disbursement_receipts
+            WHERE
+                disbursement_receipts.funding_type IN (''BC'', ''BP'')
+                AND disbursement_receipts.batch_run_date = :batchRunDate
+        )
+        SELECT
+            SUM(
+                disbursement_receipts."Full Time BC Student Loan"
+            ) AS "Full Time BC Student Loan",
```

Comment:
No need to define aliases if the column is already named as expected. Same for the other similar cases.

Reply:
I removed the aliases and send the file in the email (below). Even though we have the expected name, the sums wouldn't return the columns as aliases. I think it is because there are several sums happening and SQL expects a specific alias for each sum.
![image](https://github.com/user-attachments/assets/57233b61-8a51-46a1-8bfe-b376dd7b373e)

Reply:
Nevermind, my bad!